src/app/user/profile/profile.page.html

```diff
@@ -4,51 +4,85 @@
     <div class="heading-title">
       <h1>Профил</h1>
       <p>
-        {{user.role === "regular" ? 'на потребител' : "на организатор"}}
+        {{user?.role === "regular" ? 'на потребител' : "на организатор"}}
```

Comment:
Add the admin and be consistent with the quotes - “ ‘ “ and “ “ “

Reply:
done